python/VariableDt.ipynb

```diff
@@ -0,0 +1,4277 @@
+{
```

Comment:
4277 new lines? Has this file been cleaned before checkin?

Reply:
Probably not? Are you using ipynb_drop_output script?

Reply:
I didn't touch the file, so I don't know. If you are using that script, it probably is fine.